docs/dev/build_riscv64.md

```diff
@@ -0,0 +1,184 @@
+# Cross compile OpenVINO™ Runtime for RISCV64 systems
+This guide shows how to build OpenVINO Runtime for 64-bit RISC-V devices. Due to limited resources, cross compilation is used now for building OpenVINO targeting RISC-V development boards.
+
+Cross compilation was tested on the following hosts:
+- Ubuntu 22.04 (64-bit), x64
+
+The software was validated on the following devices:
+- [Lichee Pi 4A](https://wiki.sipeed.com/hardware/en/lichee/th1520/lp4a.html) with RVV 0.7.1
+- [Banana Pi BPI-F3](https://www.banana-pi.org/en/banana-pi-sbcs/175.html) with RVV 1.0
+
+
+## Software requirements
+
+- [CMake](https://cmake.org/download/) 3.13 or higher
+- GCC 7.5 or higher (for non-RVV) / [xuantie-gnu-toolchain](https://github.com/XUANTIE-RV/xuantie-gnu-toolchain) (for RVV)
+- Python 3.10 for OpenVINO Runtime Python API
```

Comment:
Python 3.8 or higher

Reply:
Currently only Python 3.10 is tested, and there is something wrong with Python 3.11. So I am not sure if it is suitable here to mention "Python 3.8 or higher" 🤔